src/slack/slack.js

```diff
@@ -597,29 +596,39 @@ export class Slack {
     const userMessage = order.messages[0]
     logger.debug(`running updateQuestion. userMessage: ${JSON.stringify(userMessage)}`)
 
+    let orderConfirmation
     if (userMessage) {
-      const {question, button: additionalButton} = userMessage
-      await this.updateMessage(order, {
-        actions: [additionalButton, CANCEL_ORDER_ACTION].filter(Boolean),
-        fields: this.getOrderFields(order),
-      })
+      const {channel, ts} = order.orderConfirmation
+      try {
+        await this.boltApp.client.chat.delete({channel, ts})
+      } catch (err) {
+        logger.error(`Failed to delete message on channel '${channel}': ${err}`)
+      }
 
+      const {question, button: additionalButton} = userMessage
       try {
-        await this.boltApp.client.chat.postMessage({
+        const orderAttachment = this.userOrderAttachment(order, question)
+        orderAttachment.actions = [additionalButton, CANCEL_ORDER_ACTION].filter(Boolean)
+
+        orderConfirmation = await this.boltApp.client.chat.postMessage({
           channel: userId,
-          text: question,
+          attachments: [orderAttachment],
+          text: ' ',
```

Comment:
as of this docs: https://api.slack.com/methods/chat.postMessage#args
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/6044119/155701493-ecdd479c-6d13-45ae-90cc-aaa524688655.png">
but maybe the `text` attribute won't be needed at all, if we use `fallback`? check my other comment